### PR TITLE
Change of SiStrip cross-talk parameters: improvement of the simulated hits

### PR DESCRIPTION
**Pull request by mjansova**
This PR follows a request of the strip local reco conveners (@mmusich, @echabert).

It aims to be integrated in 10_2_0_pre6.

This PR contains mainly changes in the configuration files SimGeneral/MixingModule/python/SiStripSimParameters_cfi.py and SimTracker/SiStripDigitizer/python/SiStripDigiSimLink_cfi.py. It will affect the simulation of SiStrip tracker hits leading to a better description of their properties.

The cross-talk parameters (called here "CouplingConstants") describe the charge sharing between neighbor channels in the SiStrip tracker. Those parameters are used by the file SimTracker/SiStripDigitizer/plugins/SiTrivialInduceChargeOnStrips.cc .

Thus only the shape of the simulated and then reconstructed clusters will be affected: change of the seed charge, change of the cluster width. The total cluster charge will almost be unchanged as the parameters only change the charge sharing but not the normalization.

The previous parameters were determined at run I but with ageing, these parameters had to be re-evaluated.

A measurement have been done (VR run with cosmics data) and the results are propagated into the config files.

Data/MC validation have been made to ensure that the new parameters improve the agreement of the cluster seed charge and the cluster width.

The validation plots can be found here: https://indico.cern.ch/event/688867/contributions/3040427/attachments/1670393/2679458/cross_talk2018_validation.pdf

On top of the configuration file, a modification of  SimTracker/SiStripDigitizer/plugins/SiTrivialInduceChargeOnStrips.cc has been done to allow the activation of the new parameters through flags.

They are split into two categories: barrel modules (activated via  CouplingConstantsRunIIDecB ) and disk/wheels (via CouplingConstantsRunIIDecW ). If the two booleans are set to False, the default simulation will be executed.

By default the flags are turned to False, thus the simulation will not be modified.

We plan to request special reval with the flag turned to True so that we can validate the changes and turn it to True in the next pre-realease.

### CODE REVIEW

**Comment by fabiocos** [comment]
please test

File: SimTracker/SiStripDigitizer/python/SiStripDigiSimLink_cfi.py [hunk 1]
```diff
@@ -50,6 +55,27 @@
                                CouplingConstantPeakW5  = cms.vdouble(0.968, 0.016),
                                CouplingConstantPeakW6  = cms.vdouble(0.972, 0.014),
                                CouplingConstantPeakW7  = cms.vdouble(0.964, 0.018),
+
```
[the remark this answers is not in the document]
Comment (mjansova) [reply SimTracker/SiStripDigitizer/python/SiStripDigiSimLink_cfi.py]: Indeed, there are duplications between the two configuration files, but it was there already before my commit. Then the block you are asking about is for Peak mode, which I did not touch at all and I do not see any duplications there. Could you please clarify your question?

**Comment by fabiocos** [comment]
please test

**Comment by fabiocos** [comment]
@mjansova @boudoul are the effects the expected ones? I browsed a few histograms only, and found them in agreement with the previous ones, but of course I just randomly picked a few

@perrotta @slava77 FYI as this could impact the output of data/MC agreement

**Comment by civanch** [comment]
I agree that it is not good having list of run depending parameters duplicated in different python configuration. If not in this PR but this should be indeed improved.

**Comment by civanch** [comment]
+1

**Comment by fabiocos** [comment]
+1

@echabert thanks for the detailed discussion, I understand that this PR is doing what desired, and that this can be tested more extensively in CMSSW_10_2_0_pre6

**Pull request merged**